src/series/graph.serie.line.js

```diff
@@ -1703,10 +1702,10 @@ class SerieLine extends Serie {
 
     switch ( this.getStyle( selectionType ).lineStyle ) {
 
-      case 2:
+      case 1:
```

Comment:
cc @NPellet just to make sure it is a bug. ESLint detected the duplicate `case 3`